common/src/main/scala/net/psforever/objects/serverobject/resourcesilo/ResourceSiloControl.scala

```diff
@@ -30,7 +30,14 @@ class ResourceSiloControl(resourceSilo : ResourceSilo) extends Actor with Factio
 
   def Processing : Receive = checkBehavior.orElse {
     case ResourceSilo.Use(player, msg) =>
-      sender ! ResourceSilo.ResourceSiloMessage(player, msg, resourceSilo.Use(player, msg))
+      if(resourceSilo.Faction == PlanetSideEmpire.NEUTRAL || player.Faction == resourceSilo.Faction) {
+        resourceSilo.Zone.Vehicles.find(v => v.PassengerInSeat(player).contains(0)) match {
```

Comment:
Does this now require the player to stay inside the ANT to continue depositing NTU? Because you could get out and leave it running on live.

Reply:
Is `ResourceSilo.Use` a one-and-done message, or does it need to keep being sent to keep depositing NTU?

Reply:
Can't remember off the top of my head. I want to say it repeats, but I'd have to test it to check.

Reply:
A quick check of `WSA` suggests that the process of discharging into a silo uses a `Cancellable` timer.

Reply:
Double checked and you're right, it's one use message and then the timer kicks in. Closing.